src/Button.php

```diff
@@ -43,6 +43,26 @@ public function renderView()
     }
 
     /**
+     * Makes button into a "<a>" element with a link.
+     */
+    public function link($url)
+    {
+        $this->element = 'a';
+        if (is_string($url)) {
+            $sch = substr($url, 0, 3);
+            if ($sch === 'htt' || $sch = '://') {
```

Comment:
I'm actually going to do like this: all strings are inserted as-is. Array routed through url()